statsmodels/tsa/arima/model.py

```diff
@@ -44,19 +44,23 @@ class ARIMA(sarimax.SARIMAX):
 
     Parameters
     ----------
-    endog : array_like, optional
+    endog : array_like
         The observed time-series process :math:`y`.
     exog : array_like, optional
         Array of exogenous regressors.
-    order : tuple, optional
+    order : tuple, optioanl.
```

Comment:
sorry for my inactivity on my own PR.

There is a typo here

Reply:
Thanks - I'll get it next PR.  And thanks for the PR.